services/web-app/pages/common-mdx-errors/index.js

```diff
@@ -11,7 +11,7 @@ import Link from 'next/link'
 import { NextSeo } from 'next-seo'
 import { useContext, useEffect } from 'react'
 
-import InlineError from '@/components/inline-error/inline-error'
+import InlineError from '@/components/inline-notification/inline-notification'
```

Comment:
Will also need to change `<InlineError>`s in this page to `<MdxNotification>`s
```suggestion
import MdxNotification from '@/components/mdx-notification'
```